eth/filters/api.go

```diff
@@ -416,8 +417,23 @@ func (api *PublicFilterAPI) GetFilterChanges(id rpc.ID) (interface{}, error) {
 			f.hashes = nil
 			return returnHashes(hashes), nil
 		case LogsSubscription:
-			logs := f.logs
-			f.logs = nil
+			// not reeturn the log in the lastest block to bypass issue #232
```

Comment:
return

Reply:
Yes, this is typo. I correct it at once.

Reply:
I corrected this typo just now.